### DISCUSSION TITLE
How to submit a new Board Variant?

### DISCUSSION THREAD

**Discussion by naveenneog** [espressif/arduino-esp32#9125] [Q&A]
Hi ,

We have created a new ESP32 board , I updated the `C:\Users\<username>\AppData\Local\Arduino15\packages\esp32\hardware\esp32\2.0.11\boards` file and created a folder in a  `C:\Users\<username>\AppData\Local\Arduino15\packages\esp32\hardware\esp32\2.0.11\variants`. It is showing properly in the selection 

![image](https://github.com/espressif/arduino-esp32/assets/40742835/73cbf9cc-2062-47a3-bf9b-58b2b6f328d0)
 
 Can we raise a pull request to include that in the build, Any tags or more changes is needed for this? I am not aware of any test files needs to be updated. Please help

**Comment by VojtechBartoska** [comment]
Hi, take a look on Pull Request with some of the already added boards.

https://github.com/espressif/arduino-esp32/pull/9050
